modules/fedimint-lnv2-client/src/send_sm.rs

```diff
@@ -46,23 +45,10 @@ pub struct SendSMCommon {
     pub funding_txid: TransactionId,
     pub gateway_api: SafeUrl,
     pub contract: OutgoingContract,
-    pub invoice: Invoice,
+    pub invoice_data: LightningInvoice,
```

Comment:
Did we end up changing this from `invoice_data`?

Reply:
Just changed it.